portal/src/graphql/portal/ApplicationsConfigurationScreen.module.scss

```diff
@@ -55,3 +55,19 @@
   pointer-events: none;
   opacity: 0.5;
 }
+
+.callout {
+  padding: 8px;
+}
+
+.copyButtonRoot {
+  margin: 4px;
```

Comment:
For example, this should be referred as `className={styles.copyButton}` (No suffix Root because we do not assume the internal HTML structure of the component)